app/views/layouts/admin_layout.html.erb

```diff
@@ -0,0 +1,88 @@
+<% content_for :head do %>
```

Comment:
We should use the [layout_for_admin](https://components.publishing.service.gov.uk/component-guide/layout_for_admin) and the header/footer components instead of duplicating this code.

Reply:
Hey @alex-ju - this is using the layout_for_admin - we use the `head` yield:
https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_layout_for_admin.html.erb